test/suites/basic.sh

```diff
@@ -547,7 +548,17 @@ test_basic_usage() {
   ! lxc profile rename default foobar || false
   ! lxc profile delete default || false
 
+  # Should fail to override root device storage pool when container is already initialised.
```

Comment:
This should be:

> # Should fail to override device if it already exists in the instance.

As oppose to the container already being initialised.

Reply:
I've added test cases for both the situations we discussed and grepped for the specific error we expect.